android/gradle.properties

```diff
@@ -1,5 +1,5 @@
 customerio.reactnative.kotlinVersion=1.7.21
-customerio.reactnative.compileSdkVersion=30
-customerio.reactnative.targetSdkVersion=30
+customerio.reactnative.compileSdkVersion=33
```

Comment:
is this requirement of grade 8? or compatibility with other platforms ?

Reply:
Mainly to match with Android SDK for consistent behavior.